[2024-03-18 11:02:24][INFO][main.Main]success load log file
[2024-03-18 11:02:24][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 11:02:24][INFO][main.Main]process run!
[2024-03-18 11:02:24][INFO][main.Main]pid@hostname:19896@jwmoon
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 11:02:24][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 11:02:24][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 11:02:25][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 11:02:25][INFO][mone.db.conn.DbManager]success initialize (TiberoDB) : config/mybatis-tibero-config.xml
[2024-03-18 11:02:25][INFO][mone.schedule.scheduler.ScheduleManager][스케쥴링ON/OFF]  DB 수집 on 
[2024-03-18 11:02:25][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 11:02:25][INFO][mone.schedule.scheduler.ScheduleManager]start schedule!
[2024-03-18 11:03:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 11:03:00][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 11:03:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 11:03:00][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error querying database.  Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
### The error may exist in config/sql-sudoDB.xml
### The error may involve TiberoSudoDB.select_vw_qw_emphs_manage_area-Inline
### The error occurred while setting parameters
### SQL: SELECT    EMPHS_MANAGE_AREA_CD,    EMPHS_MANAGE_AREA_NM,    WUB_CD,    WUB_NM,    APPN_DE,    ABL_DE,    USE_YN,    USE_CD_NM,    to_char(INPUT_DT, 'YYYY-MM-DD') as INPUT_DT,    to_char(UPDT_DT, 'YYYY-MM-DD') as UPDT_DT,    RM,    CHANGE_YM,    DETAIL_MAP_ATCHMNFL_NO,    SPOT_PHOTO_ATCHMNFL_NO,    LEGALDONG_CD,    LEGALDONG_ADDR,    RN_CD,    RN_ADDR,    POPSRV,    INSPCT_CO,    INSPCT_WQY_MDAT_PRVONSH,    STD_CO   FROM    VW_QW_EMPHS_MANAGE_AREA   WHERE UPDT_DT >= to_date(?, 'YYYY-MM-DD')
### Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
[2024-03-18 11:03:05][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 11:03:05][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 11:03:05][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 11:03:05][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 11:03:05][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 11:03:05][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error querying database.  Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
### The error may exist in config/sql-sudoDB.xml
### The error may involve TiberoSudoDB.select_vw_qw_emphs_manage_area-Inline
### The error occurred while setting parameters
### SQL: SELECT    EMPHS_MANAGE_AREA_CD,    EMPHS_MANAGE_AREA_NM,    WUB_CD,    WUB_NM,    APPN_DE,    ABL_DE,    USE_YN,    USE_CD_NM,    to_char(INPUT_DT, 'YYYY-MM-DD') as INPUT_DT,    to_char(UPDT_DT, 'YYYY-MM-DD') as UPDT_DT,    RM,    CHANGE_YM,    DETAIL_MAP_ATCHMNFL_NO,    SPOT_PHOTO_ATCHMNFL_NO,    LEGALDONG_CD,    LEGALDONG_ADDR,    RN_CD,    RN_ADDR,    POPSRV,    INSPCT_CO,    INSPCT_WQY_MDAT_PRVONSH,    STD_CO   FROM    VW_QW_EMPHS_MANAGE_AREA   WHERE UPDT_DT >= to_date(?, 'YYYY-MM-DD')
### Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
[2024-03-18 11:05:42][INFO][main.Main]success load log file
[2024-03-18 11:05:42][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 11:05:42][INFO][main.Main]process run!
[2024-03-18 11:05:42][INFO][main.Main]pid@hostname:18212@jwmoon
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 11:05:42][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 11:05:42][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 11:05:42][ERROR][mone.db.conn.TiberoSudoReconnectThread]Exception tiberoSudoDbConnVal 1st
[2024-03-18 11:05:42][INFO][mone.db.conn.TiberoSudoReconnectThread](NG)disconnected db, try connection ::: TiberoSudoDB 
[2024-03-18 11:05:43][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 11:06:43][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 11:07:43][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 11:08:43][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:28:17][INFO][main.Main]success load log file
[2024-03-18 18:28:17][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 18:28:17][INFO][main.Main]process run!
[2024-03-18 18:28:17][INFO][main.Main]pid@hostname:29284@jwmoon
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 18:28:17][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 18:28:17][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 18:28:17][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:28:17][INFO][mone.db.conn.DbManager]success initialize (TiberoDB) : config/mybatis-tibero-config.xml
[2024-03-18 18:28:17][INFO][mone.db.conn.TiberoReconnectThread]Exception tiberoDbConnVal 1st
[2024-03-18 18:28:17][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:28:17][INFO][mone.schedule.scheduler.ScheduleManager][스케쥴링ON/OFF]  DB 수집 on 
[2024-03-18 18:28:17][INFO][mone.schedule.scheduler.ScheduleManager]start schedule!
[2024-03-18 18:29:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:00][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:29:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:00][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error querying database.  Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
### The error may exist in config/sql-sudoDB.xml
### The error may involve TiberoSudoDB.select_vw_qw_emphs_manage_area-Inline
### The error occurred while setting parameters
### SQL: SELECT    EMPHS_MANAGE_AREA_CD,    EMPHS_MANAGE_AREA_NM,    WUB_CD,    WUB_NM,    APPN_DE,    ABL_DE,    USE_YN,    USE_CD_NM,    to_char(INPUT_DT, 'YYYY-MM-DD') as INPUT_DT,    to_char(UPDT_DT, 'YYYY-MM-DD') as UPDT_DT,    RM,    CHANGE_YM,    DETAIL_MAP_ATCHMNFL_NO,    SPOT_PHOTO_ATCHMNFL_NO,    LEGALDONG_CD,    LEGALDONG_ADDR,    RN_CD,    RN_ADDR,    POPSRV,    INSPCT_CO,    INSPCT_WQY_MDAT_PRVONSH,    STD_CO   FROM    VW_QW_EMPHS_MANAGE_AREA   WHERE UPDT_DT >= to_date(?, 'YYYY-MM-DD')
### Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
[2024-03-18 18:29:05][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:29:05][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:29:05][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:05][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:29:05][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:05][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error querying database.  Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
### The error may exist in config/sql-sudoDB.xml
### The error may involve TiberoSudoDB.select_vw_qw_emphs_manage_area-Inline
### The error occurred while setting parameters
### SQL: SELECT    EMPHS_MANAGE_AREA_CD,    EMPHS_MANAGE_AREA_NM,    WUB_CD,    WUB_NM,    APPN_DE,    ABL_DE,    USE_YN,    USE_CD_NM,    to_char(INPUT_DT, 'YYYY-MM-DD') as INPUT_DT,    to_char(UPDT_DT, 'YYYY-MM-DD') as UPDT_DT,    RM,    CHANGE_YM,    DETAIL_MAP_ATCHMNFL_NO,    SPOT_PHOTO_ATCHMNFL_NO,    LEGALDONG_CD,    LEGALDONG_ADDR,    RN_CD,    RN_ADDR,    POPSRV,    INSPCT_CO,    INSPCT_WQY_MDAT_PRVONSH,    STD_CO   FROM    VW_QW_EMPHS_MANAGE_AREA   WHERE UPDT_DT >= to_date(?, 'YYYY-MM-DD')
### Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
[2024-03-18 18:29:10][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:29:10][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:29:10][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:10][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:29:10][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:10][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error querying database.  Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
### The error may exist in config/sql-sudoDB.xml
### The error may involve TiberoSudoDB.select_vw_qw_emphs_manage_area-Inline
### The error occurred while setting parameters
### SQL: SELECT    EMPHS_MANAGE_AREA_CD,    EMPHS_MANAGE_AREA_NM,    WUB_CD,    WUB_NM,    APPN_DE,    ABL_DE,    USE_YN,    USE_CD_NM,    to_char(INPUT_DT, 'YYYY-MM-DD') as INPUT_DT,    to_char(UPDT_DT, 'YYYY-MM-DD') as UPDT_DT,    RM,    CHANGE_YM,    DETAIL_MAP_ATCHMNFL_NO,    SPOT_PHOTO_ATCHMNFL_NO,    LEGALDONG_CD,    LEGALDONG_ADDR,    RN_CD,    RN_ADDR,    POPSRV,    INSPCT_CO,    INSPCT_WQY_MDAT_PRVONSH,    STD_CO   FROM    VW_QW_EMPHS_MANAGE_AREA   WHERE UPDT_DT >= to_date(?, 'YYYY-MM-DD')
### Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
[2024-03-18 18:29:15][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:29:15][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:29:15][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:15][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:29:15][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:15][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error querying database.  Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
### The error may exist in config/sql-sudoDB.xml
### The error may involve TiberoSudoDB.select_vw_qw_emphs_manage_area-Inline
### The error occurred while setting parameters
### SQL: SELECT    EMPHS_MANAGE_AREA_CD,    EMPHS_MANAGE_AREA_NM,    WUB_CD,    WUB_NM,    APPN_DE,    ABL_DE,    USE_YN,    USE_CD_NM,    to_char(INPUT_DT, 'YYYY-MM-DD') as INPUT_DT,    to_char(UPDT_DT, 'YYYY-MM-DD') as UPDT_DT,    RM,    CHANGE_YM,    DETAIL_MAP_ATCHMNFL_NO,    SPOT_PHOTO_ATCHMNFL_NO,    LEGALDONG_CD,    LEGALDONG_ADDR,    RN_CD,    RN_ADDR,    POPSRV,    INSPCT_CO,    INSPCT_WQY_MDAT_PRVONSH,    STD_CO   FROM    VW_QW_EMPHS_MANAGE_AREA   WHERE UPDT_DT >= to_date(?, 'YYYY-MM-DD')
### Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
[2024-03-18 18:29:17][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:29:17][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:29:20][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:29:20][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:29:20][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:20][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:29:20][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:29:20][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error querying database.  Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
### The error may exist in config/sql-sudoDB.xml
### The error may involve TiberoSudoDB.select_vw_qw_emphs_manage_area-Inline
### The error occurred while setting parameters
### SQL: SELECT    EMPHS_MANAGE_AREA_CD,    EMPHS_MANAGE_AREA_NM,    WUB_CD,    WUB_NM,    APPN_DE,    ABL_DE,    USE_YN,    USE_CD_NM,    to_char(INPUT_DT, 'YYYY-MM-DD') as INPUT_DT,    to_char(UPDT_DT, 'YYYY-MM-DD') as UPDT_DT,    RM,    CHANGE_YM,    DETAIL_MAP_ATCHMNFL_NO,    SPOT_PHOTO_ATCHMNFL_NO,    LEGALDONG_CD,    LEGALDONG_ADDR,    RN_CD,    RN_ADDR,    POPSRV,    INSPCT_CO,    INSPCT_WQY_MDAT_PRVONSH,    STD_CO   FROM    VW_QW_EMPHS_MANAGE_AREA   WHERE UPDT_DT >= to_date(?, 'YYYY-MM-DD')
### Cause: java.sql.SQLException: JDBC-8033:Specified schema object was not found.
at line 25, column 5 of null:
			VW_QW_EMPHS_MANAGE_AREA
			^
[2024-03-18 18:31:04][INFO][main.Main]success load log file
[2024-03-18 18:31:04][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 18:31:04][INFO][main.Main]process run!
[2024-03-18 18:31:04][INFO][main.Main]pid@hostname:9560@jwmoon
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 18:31:04][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 18:31:04][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 18:31:04][ERROR][mone.db.conn.TiberoSudoReconnectThread]Exception tiberoSudoDbConnVal 1st
[2024-03-18 18:31:04][INFO][mone.db.conn.TiberoSudoReconnectThread](NG)disconnected db, try connection ::: TiberoSudoDB 
[2024-03-18 18:31:04][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 18:32:04][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:33:04][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:35:04][INFO][main.Main]success load log file
[2024-03-18 18:35:04][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 18:35:04][INFO][main.Main]process run!
[2024-03-18 18:35:04][INFO][main.Main]pid@hostname:16304@jwmoon
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 18:35:04][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 18:35:04][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 18:35:04][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:35:04][INFO][mone.db.conn.DbManager]success initialize (TiberoDB) : config/mybatis-tibero-config.xml
[2024-03-18 18:35:04][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:36:04][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:36:04][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:37:04][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:37:04][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:38:04][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:38:04][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:39:04][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:39:04][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:40:04][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:40:04][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:42:14][INFO][main.Main]success load log file
[2024-03-18 18:42:14][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 18:42:14][INFO][main.Main]process run!
[2024-03-18 18:42:14][INFO][main.Main]pid@hostname:25032@jwmoon
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 18:42:14][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 18:42:14][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 18:42:14][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:42:14][INFO][mone.db.conn.DbManager]success initialize (TiberoDB) : config/mybatis-tibero-config.xml
[2024-03-18 18:42:14][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:42:14][INFO][mone.schedule.scheduler.ScheduleManager][스케쥴링ON/OFF]  DB 수집 on 
[2024-03-18 18:42:14][INFO][mone.schedule.scheduler.ScheduleManager]start schedule!
[2024-03-18 18:43:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:00][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:43:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:00][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:43:05][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:43:05][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:43:05][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:05][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:43:05][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:05][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:43:10][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:43:10][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:43:10][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:10][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:43:10][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:10][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:43:14][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:43:14][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:43:15][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:43:15][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:43:15][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:15][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:43:15][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:15][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:43:20][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:43:20][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:43:20][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:20][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:43:20][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:20][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:43:25][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:43:25][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:43:25][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:25][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:43:25][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:25][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:43:30][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:43:30][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:43:30][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:30][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 18:43:30][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:43:30][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:54:29][INFO][main.Main]success load log file
[2024-03-18 18:54:29][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 18:54:29][INFO][main.Main]process run!
[2024-03-18 18:54:29][INFO][main.Main]pid@hostname:27640@jwmoon
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 18:54:29][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 18:54:29][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 18:54:29][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:54:29][INFO][mone.db.conn.DbManager]success initialize (TiberoDB) : config/mybatis-tibero-config.xml
[2024-03-18 18:54:30][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:55:29][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:55:30][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:56:11][INFO][main.Main]success load log file
[2024-03-18 18:56:11][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 18:56:11][INFO][main.Main]process run!
[2024-03-18 18:56:11][INFO][main.Main]pid@hostname:18188@jwmoon
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 18:56:11][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 18:56:11][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 18:56:11][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:56:11][INFO][mone.db.conn.DbManager]success initialize (TiberoDB) : config/mybatis-tibero-config.xml
[2024-03-18 18:56:11][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:56:11][INFO][mone.schedule.scheduler.ScheduleManager][스케쥴링ON/OFF]  DB 수집 on 
[2024-03-18 18:56:12][INFO][mone.schedule.scheduler.ScheduleManager]start schedule!
[2024-03-18 18:57:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:00][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2023-01-01
[2024-03-18 18:57:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:00][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:57:05][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:57:05][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:57:05][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:05][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2023-01-01
[2024-03-18 18:57:05][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:05][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:57:10][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:57:10][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:57:10][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:10][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2023-01-01
[2024-03-18 18:57:10][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:10][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:57:11][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 18:57:11][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 18:57:15][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:57:15][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:57:15][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:15][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2023-01-01
[2024-03-18 18:57:15][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:15][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:57:20][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:57:20][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:57:20][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:20][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2023-01-01
[2024-03-18 18:57:20][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:20][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 18:57:25][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 전:false
[2024-03-18 18:57:25][ERROR][mone.schedule.job.CollectData]즉시 재시도 여부 설정 후 :true
[2024-03-18 18:57:25][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:25][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2023-01-01
[2024-03-18 18:57:25][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 18:57:25][ERROR][mone.schedule.job.CollectData]PersistenceException
### Error updating database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for TiberoDB.insert_vw_qw_emphs_manage_area_test
[2024-03-18 19:05:56][INFO][main.Main]success load log file
[2024-03-18 19:05:56][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 19:05:56][INFO][main.Main]process run!
[2024-03-18 19:05:56][INFO][main.Main]pid@hostname:16360@jwmoon
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 19:05:56][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 19:05:56][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 19:05:56][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 19:05:56][INFO][mone.db.conn.DbManager]success initialize (TiberoDB) : config/mybatis-tibero-config.xml
[2024-03-18 19:05:56][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 19:06:48][INFO][main.Main]success load log file
[2024-03-18 19:06:48][INFO][mone.common.conf.ConfigManager]load config complete!!
[2024-03-18 19:06:48][INFO][main.Main]process run!
[2024-03-18 19:06:48][INFO][main.Main]pid@hostname:32840@jwmoon
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization...
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]Using logger: null
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.debug.stack.prefix is not defined
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.warn.threshold is not defined
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.sqltiming.error.threshold is not defined
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.dump.booleanastruefalse is not defined (using default value false)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.maxlinelength is not defined (using default of 90)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.dump.fulldebugstacktrace is not defined (using default value false)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.statement.warn is not defined (using default value false)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.select is not defined (using default value true)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.insert is not defined (using default value true)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.update is not defined (using default value true)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.delete is not defined (using default value true)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.create is not defined (using default value true)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.dump.sql.addsemicolon is not defined (using default value false)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.auto.load.popular.drivers is not defined (using default value true)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.drivers is not defined
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql is not defined (using default value true)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.trim.sql.extrablanklines is not defined (using default value true)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]x log4jdbc.suppress.generated.keys.exception is not defined (using default value false)
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]log4jdbc-logj2 properties initialization done.
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]DriverSpy intialization...
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]WARNING!  log4jdbc couldn't find any underlying jdbc drivers.
[2024-03-18 19:06:48][DEBUG][log4jdbc.debug]DriverSpy intialization done.
[2024-03-18 19:06:48][INFO][mone.db.conn.DbManager]success initialize (TiberoSudoDB) : config/mybatis-tiberoSudo-config.xml
[2024-03-18 19:06:48][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 19:06:48][INFO][mone.db.conn.DbManager]success initialize (TiberoDB) : config/mybatis-tibero-config.xml
[2024-03-18 19:06:48][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 19:06:48][INFO][mone.schedule.scheduler.ScheduleManager][스케쥴링ON/OFF]  DB 수집 on 
[2024-03-18 19:06:48][INFO][mone.schedule.scheduler.ScheduleManager]start schedule!
[2024-03-18 19:07:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 19:07:00][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2023-01-01
[2024-03-18 19:07:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 19:07:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_wqmsm, SelectEndDt=null}
[2024-03-18 19:07:00][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 Detail 조회 시작일 2023-01-01
[2024-03-18 19:07:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_wqmsm, SelectEndDt=null}
[2024-03-18 19:07:48][DEBUG][mone.db.conn.TiberoSudoReconnectThread](OK)db connected  ::: TberoSudoDB 
[2024-03-18 19:07:48][INFO][mone.db.conn.TiberoReconnectThread](OK)db connected  ::: TiberoDB 
[2024-03-18 19:08:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 19:08:00][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 DB 조회 시작일 2024-02-16
[2024-03-18 19:08:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_emphs_manage_area, SelectEndDt=null}
[2024-03-18 19:08:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_wqmsm, SelectEndDt=null}
[2024-03-18 19:08:00][INFO][mone.schedule.job.CollectData]:::::::::::: 중점관리지역 Detail 조회 시작일 2023-12-10
[2024-03-18 19:08:00][INFO][mone.schedule.job.CollectData]{SelectFromDt=20230101, chkTablenm=vw_qw_wqmsm, SelectEndDt=null}
